bundles/CoreBundle/Command/ThumbnailsImageCommand.php

```diff
@@ -65,13 +75,29 @@ protected function configure()
                 null,
                 InputOption::VALUE_NONE,
                 'do not generate high-res (@2x) versions of thumbnails'
+            )->addOption(
+                'processes',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The number of parallel processes to run',
+                1
+            )->addOption(
+                'child',
```

Comment:
This options seems to be useless?

Reply:
used by Parallelization trait. marked it "for internal use"